oe_authentication.module

```diff
@@ -16,7 +16,7 @@ use Drupal\Core\Field\BaseFieldDefinition;
  * Remove the option to delete users while keeping the option to block them.
  */
 function oe_authentication_user_cancel_methods_alter(&$methods) {
-  if (\Drupal::currentUser()->id() == 1) {
+  if (\Drupal::currentUser()->hasPermission('administer users')) {
```

Comment:
We cannot use this permission as we assign it to some roles but we don't want them to be able to do those type of actions. Let's create a permission "delete users" and use that.

Reply:
@brummbar yeh, I was askin' this in #79

Reply:
My bad, missed the notification :(